modules/nickserv.cpp

```diff
@@ -119,9 +141,18 @@ class CNickServ : public CModule {
             MCString msValues;
             msValues["password"] = GetNV("Password");
             PutIRC(CString::NamedFormat(GetNV("IdentifyCmd"), msValues));
+        } else if(Nick.NickEquals(sNickServName) && sMessage.find("Password accepted") != CString::npos && m_bJoinAfterIdentified) {
```

Comment:
Atheme uses "You are now identified" (https://github.com/atheme/atheme/blob/4fd9583b7b8498fb5379b2c07ec4de50f877d85e/modules/operserv/identify.c#L85)